ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```diff
@@ -125,6 +125,11 @@ def get_connection_attributes(self, connection):
         self.name = self.database_component
         self.database = self.connection_props.get("database", "")
         if self.database:
+            # PyMySQL encodes names with utf-8
+            try:
+                self.database = self.database.decode()
```

Comment:
Just a suggestion, depending on the contents of `self.database` it may be a good idea to set the `errors` argument in `decode` to something else than the default [`strict`](https://docs.python.org/3/library/stdtypes.html#bytes.decode).

Also, if it is expected that some attribute won't be present, then it is better to handle this with `hasattr` instead of with a `try` that should be reserved for actually exceptional situations.